src/superface-test.ts

```diff
@@ -94,12 +104,30 @@ export class SuperfaceTest {
     );
     const processRecordings = options?.processRecordings ?? true;
     const inputVariables = searchValues(testCase.input, options?.hideInput);
+    let recordingsType: RecordingType;
```

Comment:
if we change the `prepare` and `teardown` property this part would be also simplified to something like: `const recordingsType = options?.recordingsType ?? RecordingType.MAIN`